app/features/login/views/login_fastlane_ci.erb

```diff
@@ -15,7 +15,7 @@
           <hr />
 
           <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-            <input class="mdl-textfield__input" type="text" name="email" id="email">
```

Comment:
Is that the only field we have to auto-fill? I think we ask for 2 API tokens, one for the bot, one for the user, right? I guess we could apply this for both

Reply:
I'm not aware of more than the environment variable with the initial clone token to be used after the onboarding